### DISCUSSION TITLE
Which is the best way to monitorate the force during a crash test?

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1725] [Q&A] [answered]
Hi, i need to run a crash test of a component compressed trhought a rigid part with a defined load curve (displacement line profile) and evaluate the peak force during the crash test. The wall and the component are joined together through an extra node boundary.
At the moment I'm evaluating the force through the /TH/RBODY card. Is this the best way to procede?

**Comment by SyazwanSamad** [comment, marked as answer]
The current method that you're using is basically similar to physical tests where the force is taken from load transducer. I think it should be fine

**Comment by MarianBulla** [comment]
Hi FGarelli,
yes, I agree with SyazwanSamad. You can keep an eye on the T01 file, while the calculation is ongoing.

You can track the forces from the RBody or sections.
Be careful with forces. They are stored as impulse. So take the 1st derivate (by time) of it to obtain the force.

Hope it helps you ?

Thanks and best regards,
Marian
